rasgotransforms/rasgotransforms/transforms/metric/metric.yaml

```diff
@@ -0,0 +1,56 @@
+name: metric
+type: insight
+tags:
+  - insight
+description: Calculate metric values given a metric definition
+arguments:
+  timestamp:
```

Comment:
Is naming a column argument "timestamp" confusing?  Would something like "time_column" make more sense?